lessons/Data-Science/AI/ai-training/langs/en-us/index.adoc

```diff
@@ -0,0 +1,348 @@
+= Training Artificial Intelligence
+
+@description{Students consider what training is by exploring two unique examples: song recommendation and plagiarism detection. As a result of this exploration, they learn that training, a resource-intensive and time-intensive process, is the act of transforming data into a model.
+}
+
+@ifproglang{pyret}{
+@lesson-prereqs{ai-data-driven-algorithms}
+}
+
+@keywords{}
+
+[@lesson-intro-table]
+|===
+| Lesson Goals
+| Students will be able to...
+
+@objectives
+
+| Student-facing Lesson Goals
+|
+
+Let's think about training, the act of transforming data into a model.
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+
+== Song Recommendation Systems
+
+@objective{training}
+
+=== Overview
+
+Students explore song recommendation, another example of data-driven algorithms at work. Students consider how data aggregation is a key component of machine learning.
+
+=== Launch
+
+Data is at the heart of data science, and--as you learned during the previous lesson--@vocab{data-driven algorithms} are at the heart of AI. This statement is true of spell checkers, which we explored, along with many other types of programs that employ machine learning.
+
+While it is true that providing _more_ data often produces better results, we must also consider how other variables can influence the quality of output.
+
+@lesson-instruction{
+- Read @printable-exercise{case-study-michelle.adoc} and respond to the questions, providing as much detail as you can.
+}
+
+@teacher{Invite students to share their responses.
+
+If their responses highlight that data-driven algorithms produce a higher quality output when we provide more data--great! Your students understood the key take-away from @lesson-link{ai-data-driven-algorithms, "the previous lesson"}.
+
+You probably noticed that the solutions on @printable-exercise{case-study-michelle.adoc} bring new complexity to the conversation by acknowledging the possibility that *programmer interference* could be the cause of Michelle's increased satisfaction with her play list.
+
+If your students do *not* propose that the Spotify's algorithm was updated (as we suggest in the answer key), that's okay! *There's no need to reveal that possibility immediately.* We recommend moving on with the lesson. After completing @printable-exercise{song-recommending.adoc}, you can circle back to @printable-exercise{case-study-michelle.adoc}.
+}
+
+=== Investigate
+
+For Michelle to get better results, changing the code is not necessarily needed... although it is always a possibility! Let's consider song recommendation in more depth, and explore the idea that sometimes, _there is a need for algorithms to change_.
+
+Very broadly, a song recommendation system does two things:
+
+- it has some knowledge of what the listener already likes
```

Comment:
This is not a thing that song recommendation systems do. It is something they work with.